### DISCUSSION TITLE
Systemd-boot/UKI Support

### DISCUSSION THREAD

**Discussion by roshan20039** [ostreedev/ostree#3148] [General]
1) Does OSTree support systemd-boot?
2) Is yes, does it support Type2, Uki image

What is the status of the above two items currently?

**Comment by ericcurtin** [comment]
Both are in progress, see:

https://github.com/ostreedev/ostree/issues/1719

and

https://github.com/ostreedev/ostree/issues/2753

note the 2 features spoken about are not mutually exclusive. I see UKI support being worked on in grub, while there is also some ongoing work to support systemd-boot.

Both probably need more hands of deck to be further accelerated.
